### ISSUE TITLE
Travis CI fail: error: comparison between signed and unsigned integer expressions

### ISSUE THREAD

**Issue by Vincent43** [netblue30/firejail#3174]
Since https://github.com/netblue30/firejail/commit/80b50300b8349a5b47a9ba05cd2a8f9a4f1c17c5 the [Travis CI](https://travis-ci.org/netblue30/firejail/builds/639986140) show following errors:
```
sandbox.c: In function ‘monitor_application’:

sandbox.c:340:15: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]

       if (pid == dhclient4_pid || pid == dhclient6_pid)

               ^

sandbox.c:340:39: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]

       if (pid == dhclient4_pid || pid == dhclient6_pid)

                                       ^

cc1: all warnings being treated as errors

make[1]: *** [sandbox.o] Error 1

make[1]: Leaving directory `/home/travis/build/netblue30/firejail/src/firejail'

make: *** [src/firejail] Error 2

The command "( ./configure --enable-fatal-warnings --prefix=/usr && make && sudo make install && make test-travis )" exited with 2.
```
It doesn't look related to aforementioned commit though.

**Comment by rusty-snake** [comment]
https://travis-ci.org/netblue30/firejail/builds shows me https://github.com/netblue30/firejail/commit/94d018cec4daa4d409877640277156d2038628ed as first broken commit.

**Comment by kris7t** [comment]
Build is fixed by 93cecca281c42fe7b96605b612f3c206d528cae2
